metrix-mapping/src/main/java/com/powsybl/metrix/mapping/EquipmentVariable.java

```diff
@@ -73,17 +85,31 @@ static MappingVariable parseJson(JsonParser parser) throws IOException {
     private static final Set<EquipmentVariable> GENERATOR_VARIABLES = EnumSet.of(EquipmentVariable.targetP,
                                                                                  EquipmentVariable.targetQ,
                                                                                  EquipmentVariable.minP,
-                                                                                 EquipmentVariable.maxP);
+                                                                                 EquipmentVariable.maxP,
+                                                                                 EquipmentVariable.voltageRegulatorOn,
+                                                                                 EquipmentVariable.targetV);
     private static final Set<EquipmentVariable> HVDC_LINE_VARIABLES = EnumSet.of(EquipmentVariable.activePowerSetpoint,
                                                                                  EquipmentVariable.minP,
-                                                                                 EquipmentVariable.maxP);
+                                                                                 EquipmentVariable.maxP,
+                                                                                 EquipmentVariable.nominalV);
     private static final Set<EquipmentVariable> LOAD_VARIABLES = EnumSet.of(EquipmentVariable.p0,
                                                                             EquipmentVariable.q0,
                                                                             EquipmentVariable.fixedActivePower,
                                                                             EquipmentVariable.variableActivePower,
                                                                             EquipmentVariable.fixedReactivePower,
                                                                             EquipmentVariable.variableReactivePower);
-    private static final Set<EquipmentVariable> PST_VARIABLES = EnumSet.of(EquipmentVariable.currentTap);
+    private static final Set<EquipmentVariable> PHASE_TAP_CHANGER_VARIABLES = EnumSet.of(EquipmentVariable.phaseTapPosition,
+                                                                                         EquipmentVariable.regulationMode);
+    private static final Set<EquipmentVariable> TRANSFORMER_VARIABLES = EnumSet.of(EquipmentVariable.ratedU1,
```

Comment:
Maybe you should rename this variable to distinguish 2 and 3 windings transformers